engines/onnxruntime/onnxruntime-engine/build.gradle

```diff
@@ -24,7 +25,7 @@ processResources {
                 "text_embedding",
                 "token_classification",
         ]
-        def prefix = "${projectDir}/build/classes/java/main/nlp"
+        def prefix = "${baseResourcePath}/nlp"
```

Comment:
you can move this definition to line 18 and then re-use it to specify `outputs.dir`

Reply:
Thanks, updated